crates/languages/src/markdown-inline/highlights.scm

```diff
@@ -1,6 +1,20 @@
-(emphasis) @emphasis
-(strong_emphasis) @emphasis.strong
-(code_span) @text.literal
-(link_text) @link_text
-(link_label) @link_text
-(link_destination) @link_uri
+(emphasis) @markup.emphasis
```

Comment:
These selectors aren't going in the right direction.

We do partial matching on selectors, so we want them to go from less specific -> more specific.

So for this one it should be `@emphasis.markup`.

This would allow for it to be styled just by `emphasis`, but overridden by `emphasis.markup`.